README.md

```diff
@@ -63,6 +63,41 @@ accelerate launch examples/simulacra.py
 python -m trlx.sweep --config configs/sweeps/ppo_sweep.yml examples/ppo_sentiments.py
 ```
 
+## Logging
+
+trlX uses the standard Python `logging` library to log training information to the console. The default logger is set to the `INFO` level, which means that `INFO`, `WARNING`, `ERROR`, and `CRITICAL` level messages will be printed to standard output.
+
+To change the log level, you can use one of the direct setters. For example, to set the log level to `WARNING` you can use:
+
+```python
+import trlx
+
+trlx.logging.set_verbosity_warning()
+```
+
+This will suppress `INFO` level messages, but still print `WARNING`, `ERROR`, and `CRITICAL` level messages.
+
+You can also control logging verbosity by setting the `TRLX_VERBOSITY` environment variable to one of the standard logging [level names](https://docs.python.org/3/library/logging.html#logging-levels):
```

Comment:
is it `TRLX_VERBOSITY` or `TRLX_LOG_LEVEL` ?
I kind of like more `TRLX_LOG_LEVEL`